pkgs/development/r-modules/generate-shell.nix

```diff
@@ -5,13 +5,18 @@ stdenv.mkDerivation {
 
   buildCommand = "exit 1";
 
-  buildInputs = [ wget ];
+  buildInputs = [
+    wget
+    cacert
+    nix
```

Comment:
Aren't cacert and nix part of the stdenv?

Reply:
I couldn't get `generate-R-packages.R` to run without `cacert` and `nix`. 

`nix-shell generate-shell.nix` had access to the system packages, so SSL calls worked, but I hadn't updated NixOS in  a little while, and there was a mismatch in GLIBC, so `nix-hash` would not run.

`nix-shell --pure generate-shell` couldn't access the system certificates, so network calls failed due to missing SSL certificates. I think `nix-hash` was missing too. I'll check, but it might be a little while, I recently gc'd my nix store.

Reply:
Is there a better way to bring those packages into a nix-shell?

Reply:
Looking back at my commit messages, I did test `nix-shell --pure generate-shell` without `nix` and calls to `nix-hash` failed, saying it couldn't be found.

Reply:
> Aren't cacert and nix part of the stdenv?

They are not. I'm a little surprised wget doesn't link to cacert as a fallback though.

Reply:
I cannot make the shell work without these changes, and @jbedo has confirmed that this is expected behaviour in Nix. Getting cacert and nix into stdenv is obviously outside the scope of this PR. Marking this as resolved.